ArcGISRuntimeSDKQt_QMLSamples/Layers/ExportVectorTiles/ExportVectorTiles.qml

```diff
@@ -0,0 +1,275 @@
+            initialViewpoint: ViewpointCenter {
+                id: viewpoint
+                Point {
+                    x: -117.181
+                    y: 34.049
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                targetScale: 1e4
+            }
+        }
+
+
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+
+            Graphic {
+                id: exportAreaGraphic
+                symbol: SimpleLineSymbol {
+                    style: Enums.SimpleLineSymbolStyleSolid
+                    color: "green"
+                    width: 3
+                }
+            }
+        }
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    Rectangle {
+        id: exportProgressWindow
+        anchors.centerIn: parent
+
+        color: "white"
+        visible: exportJobStatus !== 0 && exportJobStatus !== 3 && exportJobStatus !== 4
+
+        border {
+            color: "black"
+            width: 2
+        }
+
+        Column {
+            id: exportProgressColumn
+            anchors.centerIn: parent
+            spacing: 10
+
+            BusyIndicator {
+                id: busyIndicator
+                anchors.horizontalCenter: parent.horizontalCenter
+            }
+
+            Text {
+                id: statusText
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: "Export job status: " + ["Not started", "Started", "Paused", "Succeeded", "Failed", "Cancelling"][exportJobStatus]
+                font.pixelSize: 16
+            }
+
+            Text {
+                id: progressText
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: exportProgress + "% Completed"
+                font.pixelSize: 16
+            }
+
+            onWidthChanged: {
+                exportProgressWindow.width = exportProgressColumn.width + 20
+            }
+
+            onHeightChanged: {
+                exportProgressWindow.height = exportProgressColumn.height + 20
+            }
+        }
+    }
+
+    // Create an extent rectangle for selecting the offline area
+    Rectangle {
+        id: extentRectangle
+        anchors.centerIn: parent
+
+        width: parent.width - parent.width * 0.1
+        height: parent.height - parent.height * 0.25
+        color: "transparent"
+        border {
+            color: "red"
+            width: 3
+        }
+
+        visible: exportJobStatus === 0 || exportJobStatus === 4
+    }
+
+    // Button to start the download
+    Button {
+        id: button
+        anchors {
+            bottom: parent.bottom
+            bottomMargin: rootRectangle.height * .05
+            horizontalCenter: parent.horizontalCenter
+        }
+        width: 150
+
+        text: "Export area"
+
+        onClicked: {
+            switch(exportJobStatus) {
```

Comment:
Same comment as the C++ implementation. Could we use an enum instead of commenting what each case number implies?